tutorials/i18n/localization_using_gettext.rst

```diff
@@ -85,6 +85,41 @@ the translated string.
 The ``msgstr`` value in PO template files (``.pot``) should **always** be empty.
 Localization will be done in the generated ``.po`` files instead.
 
+Creating the PO template (POT) using pybabel
+--------------------------------------------
+
+The Python tool pybabel has support for Godot and can be used to automatically
+create and update the POT file from your scene files and scripts.
+
+After installing ``babel`` and ``babel-godot``, for example using pip:
+
+::
+
+    pip install babel babel-godot
+
+write a mapping file (for example ``babelrc``) which will indicate which files
+pybabel needs to process (note that we process GDScript as Python, which is
+generally sufficient):
+
+::
+
+    [python: **.gd]
+    encoding = utf-8
+
+    [godot_scene: **.tscn]
+    encoding = utf-8
+
+You can then run pybabel like so:
+
+::
+
+    pybabel extract -F babelrc -k text -k LineEdit/placeholder_text -k tr -o godot-l10n.pot .
+
+Use the ``-k`` option to specify what needs to be extracted. In this case,
+arguments to :ref:`tr() <class_Object_method_tr>` will be translated, as well
+as properties named "text" (commonly used by Control nodes) and LineEdit's
+"placeholder_text" property.
```

Comment:
Out of curiosity, does pybabel offer a way for `babelrc` to define the list of functions/properties should be considered as localizable? There are many more properties in Godot which are automatically localized.

Reply:
pybabel has separate keywords/options, and keywords can only be specified on the command-line. But I can move those to an option instead.

Are you saying there's an official list of the properties of built-in nodes that should be localized? I could put this in the plugin, so there'd be no need for the user to list them.

Reply:
There doesn't seem to be :(

Reply:
We can merge this pull request for now, I'll look at documenting the list of automatically-translated properties later.